tapdb/assets_common.go

```diff
@@ -494,7 +502,8 @@ func parseGroupKeyInfo(tweakedKey, rawKey, genesisSig []byte,
 		PubKey: untweakedKey,
 	}
 
-	groupSig, err := schnorr.ParseSignature(genesisSig)
+	// TODO(jhb): make generic for non-sig witnesses
+	groupSig, err := asset.ParseGroupSig(witness)
```

Comment:
We'd then modify `GroupKey` to include both the reveal information and raw witness, no? Then that way given that you can verify the minting if needed. May depend on the actual call site though.

Reply:
When building on top of this PR I found at least the reveal information to be useful to be contained in the `GroupKey` struct.

Reply:
True, I guess it should sort of track the group key view. Will make them optional though, not sure if we have a call site where these would not be populated yet like importing the tweaked group key before having the witness or so.